xml/System.Diagnostics/StackFrameExtensions.xml

```diff
@@ -73,9 +74,9 @@
         <Parameter Name="stackFrame" Type="System.Diagnostics.StackFrame" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="stackFrame">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="stackFrame">A stack frame.</param>
+        <summary>Gets an interface pointer to the start of the native code for the method that is being executed.</summary>
+        <returns>This method always returns <see cref="F:System.IntPtr.Zero" />.</returns>
```

Comment:
Same Here, if you take a look [here](https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/src/System/Diagnostics/StackFrameExtensions.cs) you'll see that in .Net Native we actually return something else.